src/wp-content/themes/twentytwenty/functions.php

```diff
@@ -213,7 +213,7 @@ function twentytwenty_register_scripts() {
 		wp_enqueue_script( 'comment-reply' );
 	}
 
-	wp_enqueue_script( 'twentytwenty-js', get_template_directory_uri() . '/assets/js/index.js', array(), $theme_version, false );
+	wp_enqueue_script( 'twentytwenty-js', get_template_directory_uri() . '/assets/js/index.js', array(), $theme_version, array( 'in_footer' => false ) );
```

Comment:
For backward compatibility with older WordPress versions, let's remove this entirely instead since `false` is the default anyway. See https://make.wordpress.org/core/2023/07/14/registering-scripts-with-async-and-defer-attributes-in-wordpress-6-3/#:~:text=registrations%20and%20enqueues.-,Retaining%20backward%20compatibility,-For%20prior/existing for why the current change you have here is problematic in terms of BC.
```suggestion
	wp_enqueue_script( 'twentytwenty-js', get_template_directory_uri() . '/assets/js/index.js', array(), $theme_version );
```